x/escrow/time_test.go

```diff
@@ -0,0 +1,24 @@
+package escrow
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/iov-one/weave"
+)
+
+func TestIsExpired(t *testing.T) {
```

Comment:
Want to add a test with the panic?
To ensure it doesn't silently accept context with unset block time

Reply:
Addressed in https://github.com/iov-one/weave/pull/409/commits/574cd6aba8bd1708c2bfcf9ace524b85b87e6fab